bqskit/ir/gates/constant/__init__.py

```diff
@@ -14,6 +14,7 @@
 from bqskit.ir.gates.constant.cx import CXGate
 from bqskit.ir.gates.constant.cy import CYGate
 from bqskit.ir.gates.constant.cz import CZGate
+from bqskit.ir.gates.constant.ecr import ECRGate
```

Comment:
Please also add the ECRGate to the docstring in `bqskit/ir/gates/__init__.py`, so that it can be included with documentation.